components/NavBar.js

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components';
+ 
+const NavBarContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 46px 87px;
+`;
+
+const NavGroupContainer = styled.div`
+  display: flex;
+  gap: 28px;
+  align-items: center;
+`;
+
+const StyledImage = styled.img`
+  margin-right: 18px;
+`;
+
+const SimpleLinkText = styled.span`
+  font-weight: bold;
+  color: ${p => p.theme.colors.secondary};
+
+  &:hover {
+    background: ${p => p.theme.colors.primaryGradient};
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    -webkit-text-fill-color: transparent; 
+    -moz-text-fill-color: transparent;
+  }
+`;
+
+const JoinLinkInactive = styled.a`
```

Comment:
Thoughts on making this regular text elements instead of an anchor element? If someone has scrolled down the page and clicks the link their screen jumps to the top of the page, which is a bit of a jarring experience.

Reply:
good point haha got wrapped up into thinking about links, fixed!